wp-content/plugins/automatic-translation-button/plugin.php

```diff
@@ -12,6 +12,12 @@
  */
 class AutomaticTranslationButtonPlugin {
 	public function enqueue_styles_and_scripts() {
+		
+		//dont show autotranslate button if user is organisation
+		if( ! current_user_can( 'publish_pages' ) ) {
+			return false;
```

Comment:
I'm not sure if the `false` doesn't stop further enqueue propagation. Does anything stop us from `return;` without the `false`?